test/TestImport.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+module TestImport
+    ( withApp
+    , runDB
+    , authenticateAs
+
+    -- Convenience exports
```

Comment:
I thought that not using the `import as X`, `export as module X` convention would be better so you can see clearly what's being re-exported. I'm not sure though and may go back to it.